### PR TITLE
Add GHA workflow

### PR DESCRIPTION
**Pull request by archanaserver**

### CODE REVIEW

File: .github/workflows/ci_test.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    uses: theforeman/actions/.github/workflows/rubocop.yml@v0
+    with:
+        command: bundle exec rubocop --parallel --format github
+
+  test:
+    name: Tests
+    needs: rubocop
+    uses: theforeman/actions/.github/workflows/test-gem.yml@v0
+    with:
+      command: bundle exec rake test
```
Comment (adamruzicka) [inline .github/workflows/ci_test.yml:25]: This is not going to fly as there are not tests and therefore no rake task to run them
https://github.com/theforeman/hammer-cli-foreman-tasks/actions/runs/9956681232/job/27507158707?pr=57#step:5:6

**Review by adamruzicka — CHANGES_REQUESTED** [review]
I'd be fine with having the rubocop part of this

**Comment by archanaserver** [comment]
> I'd be fine with having the rubocop part of this

This make sense, I just dropped the other part. Thanks @adamruzicka.

**Review by adamruzicka — APPROVED** [review]

**Comment by adamruzicka** [comment]
Thank you @archanaserver !

**Pull request merged**